### PR TITLE
Fix: Fixed spaces causing error in --sort flag at crypto/disc/...

### PR DESCRIPTION
**Pull request by brendef**
# Description

Fixing the bug referenced at #3837 - adding a space in the sort options causes the application to produce an error. I replaced the spaces with underscores.

Before fix
beforeFix#3837

After fix
afterFix#3837

I fixed this as I like this project and would love to be more involved. 

# How has this been tested?

- [x] Iterm2 on MacOS 

To reproduce this error, reference a column name with a space when sorting on any command in the crypto/disc/ category

# Checklist:

- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).

# Others
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: openbb_terminal/cryptocurrency/discovery/dappradar_model.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 from openbb_terminal.decorators import log_start_end
 from openbb_terminal.helper_funcs import get_user_agent
 from openbb_terminal.rich_config import console
+from openbb_terminal.stocks import stocks_helper
```
Comment (jose-donato) [inline openbb_terminal/cryptocurrency/discovery/dappradar_model.py:14]: imported by mistake? can you remove it

File: openbb_terminal/cryptocurrency/discovery/dappradar_model.py [hunk 2]
```diff
@@ -64,7 +65,8 @@ def _make_request(url: str, verbose: bool = True) -> Optional[dict]:
     response = requests.get(url, headers=headers)
     if not 200 <= response.status_code < 300:
         if verbose:
-            console.print(f"[red]dappradar api exception: {response.text}[/red]")
+            console.print(
```
Comment (jose-donato) [inline openbb_terminal/cryptocurrency/discovery/dappradar_model.py:68]: formatting wrong. did you configure precommit?

**Review by jose-donato — CHANGES_REQUESTED** [review]
minor issues - lemme know if you need any help to fix those. nice job 👍

File: openbb_terminal/cryptocurrency/discovery/dappradar_model.py (hunk 1 above)
Comment (brendef) [reply openbb_terminal/cryptocurrency/discovery/dappradar_model.py]: My bad, imported by accident.

File: openbb_terminal/cryptocurrency/discovery/dappradar_model.py (hunk 2 above)
Comment (brendef) [reply openbb_terminal/cryptocurrency/discovery/dappradar_model.py]: My VS Code formatting settings weren't configured correctly, but sorted it already.

**Comment by jose-donato** [comment]
Do you need help in setting up pre-commit for the linting?

**Comment by brendef** [comment]
> 

Hey :) yes please, that would be great!

**Comment by brendef** [comment]
> Do you need help in setting up pre-commit for the linting?

@jose-donato - So I think I figured out pre-commit, everything seems to be fine

**Comment by jose-donato** [comment]
> > Do you need help in setting up pre-commit for the linting?
> 
> @jose-donato - So I think I figured out pre-commit, everything seems to be fine

It's still failing the tests

**Comment by brendef** [comment]
> > > Do you need help in setting up pre-commit for the linting?
> > 
> > 
> > @jose-donato - So I think I figured out pre-commit, everything seems to be fine
> 
> It's still failing the tests

Hey @jose-donato sorry about that. I hope that i corrected it properly this time. Let me know. I don't see any failed tests on my side.

**Pull request closed without merging**